src/main/java/com/softserveinc/ita/pageobjects/admin/MainMenu.java

```diff
@@ -95,4 +78,10 @@ public String getLoggedUserName() {
                 .should(appear, ofSeconds(5))
                 .getText();
     }
+
+    private void openPage(String pageName) {
```

Comment:
pass such param: MainMenuButtons pageName

and get name here: pageName.getPageName()

instead of repeating it in many places